libs/build_hdf5.sh

```diff
@@ -60,9 +60,9 @@ software=$name-$(echo $version | sed 's/\./_/g')
 [[ -d build ]] && rm -rf build
 mkdir -p build && cd build
 
-[[ -z $mpi ]] || extra_conf="--enable-parallel --enable-unsupported"
+[[ -z $mpi ]] || extra_conf="--enable-parallel --enable-unsupported --enable-build-mode=production"
```

Comment:
I don't know what `--enable-build-mode=production` is, but that is fine to add.

Reply:
see above.